docs/validators/validator-setup.md

```diff
@@ -167,26 +130,21 @@ gaiad query tendermint-validator-set | grep "$(gaiad tendermint show-address)"
 
 You should now see your validator in one of the Cosmos Hub explorers. You are looking for the `bech32` encoded `address` in the `~/.gaia/config/priv_validator.json` file.
 
-::: warning Note
-To be in the validator set, you need to have more total voting power than the 100th validator.
-:::
-
 ## Halting Your Validator
 
-When attempting to perform routine maintenance or planning for an upcoming coordinated
-upgrade, it can be useful to have your validator systematically and gracefully halt.
-You can achieve this by either setting the `halt-height` to the height at which
-you want your node to shutdown or by passing the `--halt-height` flag to `gaiad`.
-The node will shutdown with a zero exit code at that given height after committing
+When attempting to perform routine maintenance or planning for an upcoming coordinated upgrade, it can be useful to have your validator systematically and gracefully halt. You can achieve this by either setting the `halt-height` to the height at which you want your node to shutdown or by passing the `--halt-height` flag to `gaiad`. The node will shutdown with a zero exit code at that given height after committing
 the block.
 
+## Advanced configuration
+You can find more advanced information about running a node or a validator on the [Tendermint Core documentation](https://docs.tendermint.com/master/nodes/).
+
 ## Common Problems
 
 ### Problem #1: My validator has `voting_power: 0`
 
-Your validator has become jailed. Validators get jailed, i.e. get removed from the active validator set, if they do not vote on `500` of the last `10000` blocks, or if they double sign. 
+Your validator has become jailed. Validators get jailed, i.e. get removed from the active validator set, if they do not vote on at least `500` of the last `10000` blocks, or if they double sign. 
 
-If you got jailed for downtime, you can get your voting power back to your validator. First, if `gaiad` is not running, start it up again:
+If you got jailed for downtime, you can get your voting power back to your validator. First, if you're not using [Cosmovisor](https://docs.cosmos.network/master/run-node/cosmovisor.html) and `gaiad` is not running, start it up again:
```

Comment:
validator also need to unjail themselves `gaiad tx slashing unjail --from`

Reply:
The link to #unjail validator specifies the command :)